LFS171x/indy-material/nodejs/AgentDemoScript.md

```diff
@@ -0,0 +1,319 @@
+
+Together, that public data is what allows Faber to issue a Transcript Credential to Alice that she can later prove to Acme.
+
+Back to our story.  Faber has just set up a Hyperledger Indy Agent that can issue credentials. John Academic, Faber's most awesome transcript keeper, needs to get things working. He will have to write a Schema and Credential Definition to the Indy Ledger to do this. It is a bit complicated but nothing John Academic can't handle!
+
+
+
+1.  If you haven't already done so, go to the Faber agent tab and click **Issuing**.
+
+|![CreateSchema](images/CreateSchema.png "Publish schema to the Indy ledger")|
+|:--:|
+|*Licensed under [CC By 4.0](https://creativecommons.org/licenses/by/4.0/)*|
+
+1.  Give the schema a name, a version and a list of attributes. For example, _Transcript 1.3 with name, degree, status, year, average and social security number._
+2.  Click **Submit**.
+
+This writes the Schema to the Ledger so Faber (and others) can use it. 
+
+
+
+1.  In the Credential Definition window, select a **Schema** (Transcript 1.3), tag it and click **Submit** and wait for the action to complete (which takes a bit…).
+
+|![CreateCredDef](images/CreateCredDef.png "Create Faber Credential Definition")|
+|:--:|
+|*Licensed under [CC By 4.0](https://creativecommons.org/licenses/by/4.0/)*|
+
+John tagged his credential as MyTranscript. John probably could have come up with a better name, but hey, he's not _that_ smart.
+
+The process of creating a credential definition takes a bit of time as Indy is generating crypto keys so that the credential definition is secure and can only be used by Faber.
+
+OK - we've done the setup work. On with the story.
+
+
+### Step 3: Establishing Relationships
+
+In order for Alice to get her transcripts from her alma mater, Faber College, Alice needs to establish a digital relationship with Faber. Alice does this by sending a connection request using the Endpoint DID for Faber. In this demo, each agent's unique DID is shown at the bottom of the agent's window.
+
+> **Note: _The user interface for doing this in real agents will be MUCH better. For example, Alice might have been logged into Faber's website, been offered her transcripts and clicked a link to trigger establishing a connection. Or, she might have seen a QR code published by Faber and used the camera on her phone to start the connection. Maybe you will think of a better way for this to occur._**
+
+To create a new relationship:
+
+
+
+1.  Click **Send Connection Request**. 
+
+|![CreateRelBlank](images/CreateRelBlank.png "Blank Create Relationship Popup")|
+|:--:|
+|*Licensed under [CC By 4.0](https://creativecommons.org/licenses/by/4.0/)*|
+
+1.  Highlight and copy Faber's Endpoint DID (from the Faber tab), paste it into the "Send Connection Request" popup (above) and click **Send Connection Request**.

```

Comment:
Want to continue the above list. Indent picture above.

Reply:
Fixed